quinn/src/endpoint.rs

```diff
@@ -425,6 +429,38 @@ impl State {
     fn drive_recv(&mut self, cx: &mut Context, now: Instant) -> Result<bool, io::Error> {
         self.recv_limiter.start_cycle();
         let mut metas = [RecvMeta::default(); BATCH_SIZE];
+        let rcx = &mut ReceiveContext {
+            cx,
+            now,
+            metas: &mut metas,
```

Comment:
Could we move the definition of `metas` into `poll_socket`? One less thing to pass down that way. Reusing a PoD stack allocation probably doesn't buy us much.

Reply:
Commit 59516f7 implements this.